scripts/001-binutils.sh

```diff
@@ -8,43 +8,46 @@ BRANCH_NAME="dvp-v2.14"
 if test ! -d "$REPO_FOLDER"; then
 	git clone --depth 1 -b $BRANCH_NAME $REPO_URL && cd $REPO_FOLDER || exit 1
 else
-	cd $REPO_FOLDER && git fetch origin && git reset --hard origin/${BRANCH_NAME} || exit 1
+	cd $REPO_FOLDER && git fetch origin && git reset --hard origin/${BRANCH_NAME} && git checkout ${BRANCH_NAME} || exit 1
 fi
 
+TARGET="dvp"
+TARG_XTRA_OPTS=""
 OSVER=$(uname)
 if [ ${OSVER:0:10} == MINGW64_NT ]; then
```

Comment:
I think that we can simplify the `if` just doing 

```
if [ ${OSVER:0:5} == MINGW ]; then
        export lt_cv_sys_max_cmd_len=8000
	export CC=i686-w64-mingw32-gcc
	CFLAGS="$CFLAGS -DHAVE_DECL_ASPRINTF -DHAVE_DECL_VASPRINTF"
	TARG_XTRA_OPTS="--host=x86_64-w64-mingw32"
```

Reply:
Unfortunately, we cannot. There are 3 possible environments: MINGW32, MINGW64 and MSYS. I modified scripts only for MINGW32. Maybe someone wants to modify it also for MINGW64 but for me, it is too much. MINGW32 uses the prefix `i686` while MINGW64 uses the prefix `x86_64`.

Reply:
Of course it is possible to modify scripts to use "uname -m" for defining 32/64 bit, but I think it can be done when someone provide working solution for MINGW64

Reply:
Understood